apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/SubscriberClientImpl.SingleChannel.cs

```diff
@@ -132,6 +132,12 @@ internal RetryInfo(DateTime firstTimeOfFailureInUtc, TimeSpan? backoff = null)
         // We store the first time of error corresponding to the AckId, so that we retry only for specified duration
         // which is a requirement for exactly once delivery subscription.
         private readonly ConcurrentDictionary<string, RetryInfo> _retryableIds = new ConcurrentDictionary<string, RetryInfo>();
+
+        // This dictionary will store the Ack IDs of the newly fetched messages along with their corresponding receipt ModAck status.
+        // This is used in exactly once delivery flow only.
+        // The keys in the dictionary represents the Ack ID of the message, while value contains the receipt ModAck status.
+        // A value of null indicates that the status is not yet started or in progress. A value of true indicates success, and false indicates failure (which can be temporary or permanent).
+        private readonly ConcurrentDictionary<string, bool?> _receiptModAckStatusLookup = new ConcurrentDictionary<string, bool?>();
```

Comment:
Given that we're using a modern C# compiler, we can now just use `new()` for the RHS.